lib/Dialect/AffineOps/AffineOps.cpp

```diff
@@ -2031,6 +2041,103 @@ OpFoldResult AffineMinOp::fold(ArrayRef<Attribute> operands) {
   return results[minIndex];
 }
 
+//===----------------------------------------------------------------------===//
+// AffinePrefetchOp
+//===----------------------------------------------------------------------===//
+
+ParseResult parseAffinePrefetchOp(OpAsmParser &parser, OperationState &result) {
```

Comment:
Could we have a comment describing the syntax this function understands?

Also, consider declaring it and functions below `static`